components/bundles/bioformats_package/build.xml

```diff
@@ -6,15 +6,26 @@ Download Apache Ant from http://ant.apache.org/.
 Type "ant -p" for a list of targets.
 -->
 
-<project name="bioformats_package" default="bundle" basedir=".">
+<project name="bioformats_package" default="bundle" basedir="."  xmlns:resolver="antlib:org.apache.maven.resolver.ant">
   <description>Build file for Bio-Formats bundle</description>
   <property name="root.dir" location="../../.."/>
-  <import file="${root.dir}/ant/java.xml"/>
+  <import file="${root.dir}/ant/java.xml" />
   <property file="build.properties"/>
 
+
   <target name="docs" depends="javadoc-properties"
     description="generate the Javadocs for most components">
     <echo>----------=========== Javadocs ===========----------</echo>
+    <path id="compile.classpath"/>
```

Comment:
As mentioned in a previous version of this commit, this is equivalent to the approach I took in https://github.com/ome/bioformats/pull/3923/commits/bf8076ec08fffc2a5a2deedd6e6240a33ecc11a8#diff-fdeb882f159787e553f8d2b7f340d498880a8a54e60ae5dee163671dc9009018R203-R211 to fix the bundle generation target.